src/main/kotlin/minesweeper/BoardGenerator.kt

```diff
@@ -1,3 +1,5 @@
+package minesweeper
```

Comment:
관련된 클래스들을 패키지로 모아서 관리할 수 있도록 해보세요. 비즈니스 문제를 해결하는 클래스들은 `domain` 패키지에 위치하면 어떨까요? 아래의 글을 참고하세요.
https://edu.nextstep.camp/s/Tlh79Qon/ls/4ByDIe8q

```suggestion
package minesweeper.domain
```